### DISCUSSION TITLE
Picture Zoom

### DISCUSSION THREAD

**Discussion by Nexow91** [photoprism/photoprism#2224] [Ideas]
Hi there,

one question, would it be possible to add a feature to zoom a photo like e.g. is possible with the Windows photo app? That you use CTRL + mouse wheel to zoom in on the picture? At the moment I can only zoom the image once and here the zoom level is also fixed. I would wish for more flexibility here, otherwise the program is well thought out and fun. Only this end does not justify the means because zooming is important for my work. For example, to be able to better recognize technical / system / pipes / connections or more. That's why I'm currently using Photoprism for a better overview, but I have to use another app to zoom very close to recordings.

I look forward to your feedback and thank you for Photoprism.

**Comment by lastzero** [comment]
Will be part of the new photo viewer (which is really an epic that can't be delivered completely in one release):
- https://github.com/photoprism/photoprism/issues/1307
